pkg/dao/repository_test.go

```diff
@@ -195,19 +195,19 @@ func (suite *RepositorySuite) TestBulkCreateOneFails() {
 	t := suite.T()
 	tx := suite.tx
 
-	orgID := "1"
-	accountID := "2"
+	orgID := orgIdTest
```

Comment:
should this orgID, and any remaining hard-coded orgIDs, be `seeds.RandomOrgId()`?  I know it's very unlikely, but what if the existing database has a repository the orgID `1` and URL `repo_1_url` already?

My understanding is, since the tests no longer wipe the database first, we depend on the OrgID to keep the test data separate.